experiment/simulation/js/main.js

```diff
@@ -1 +1,268 @@
 //Your JavaScript goes in here
+
+document.addEventListener('DOMContentLoaded', function(){
+
+	var play = true
```

Comment:
Do not use var for variable declaration. Use const preferably and let where const is not possible. This comment is valid for all the JS code